Common/include/CConfig.hpp

```diff
@@ -5335,6 +5338,17 @@ class CConfig {
    */
   TURBO_PERF_KIND GetKind_TurboPerf(unsigned short val_iZone) const { return Kind_TurboPerf[val_iZone]; };
 
+  /*!
+   * \brief gets interface kind for an interface marker in turbomachinery problem
+   * \return interface kind
+   */
+  TURBO_INTERFACE_KIND GetKind_TurboInterface(unsigned short interfaceIndex) const { return Kind_TurboInterface[interfaceIndex]; }
+
+  /*!
+   * \brief Sets marker kind for an interface marker in turbomachinery problem
+   */
+  void SetKind_TurboInterface(unsigned short interfaceIndex, TURBO_INTERFACE_KIND TurboInterfaceKind) {Kind_TurboInterface[interfaceIndex] = TurboInterfaceKind ;}
```

Comment:
Do you need the Set?

Reply:
Yes it is used during the config parsing

Reply:
If that parsing is done in CConfig you can just access Kind_TurboInterface directly